modules/olm-webhook-considerations.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-webhooks.adoc
+
+[id="olm-webhook-considerations_{context}"]
+= Webhook considerations
+
+When developing an admission webhook to be managed by OLM, consider the
+following constraints:
+
+[discrete]
+[id="olm-webhook-ca_{context}"]
+=== Certificate authority constraints
+
+OLM is configured to provide each Deployment with a single certificate authority
+(CA). The logic that generates and mounts the CA into the Deployment was
+originally used by the API Service lifecycle logic. As a result:
```

Comment:
Should "Service" be lowercase?

Reply:
Meant for this to be "APIService"; it's referring to these: https://docs.openshift.com/container-platform/4.4/operators/operator_sdk/osdk-generating-csvs.html#osdk-apiservices_osdk-generating-csvs